src/spi.rs

```diff
@@ -210,6 +229,7 @@ pub mod config {
                 write_only: false,
                 cs_active_high: false,
                 duplex: Duplex::Full,
+                chunk_size: Chunk::SixtyFourBits,
```

Comment:
Should we change the default chunk size to 8bits here to be most compatible with other devices by default?

Reply:
Ah, I see. I think if it was 64bits before then let's leave it as that.

Reply:
technically it was a non chunked version before. so i think it send the complete buffer in one go up to max_transfer_size -> and because this wrapper driver was written in a way to only allow multiple of 8 clk's it was 64*8 so 64 byte... so to be fully backwards  compatible i should add a 9nth enum element Chunk::None -> that corresponds to 64. Thats the limit for the non dma buffer for the idf driver iirc.